test/jit/test_attr.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from torch.testing import FileCheck
+from torch.testing._internal.jit_utils import JitTestCase
+from torch import jit
+import unittest
+import os
+import sys
+import torch
+import torch.testing._internal.jit_utils
+
+
+if __name__ == '__main__':
+    raise RuntimeError("This test file is not meant to be run directly, use:\n\n"
+                       "\tpython test/test_jit.py TESTNAME\n\n"
+                       "instead.")
+
+
+
+class Shape:
+    def __init__(self):
+        self.center = 1.0
+
+
+class TestSetAttr(JitTestCase):
+    def test_setattr_with_default(self):
+        def func(x):
+            s = Shape()
+            y = getattr(s, "center")
+            w : list[float] = [1.0]
+            z = getattr(s, "missing", w)
+            z.append(y)
+            return z
+
+
+        graph = torch.jit.script(func).graph
+        print(graph)
+
+        # The "center" attribute is  valid on a shape object, so expect that
+        FileCheck().check("prim::GetAttr[name=\"center\"]").run(graph)
+        # The "missing" attribute does not exist, so there should be no corresponding GetAttr in AST
+        FileCheck().check_not("missing").run(graph)
+        # instead the getattr call will emit the default value, which is a list with one float element
+        FileCheck().check("float[] = prim::ListConstruct").run(graph)
```

Comment:
is there a way to check how many elements are in the list? I think that would be better test

Reply:
Now I verify that an append call is applied to the list in the IR (see line 36). Not clear how to check how many elements are exactly in the list.